src/app/client/package-lock.json

```diff
@@ -1,15743 +0,0 @@
-{
```

Comment:
Do not delete package-lock.json from repo as without the dependency references build time will be long.